src/pymor/analyticalproblems/functions.py

```diff
@@ -372,7 +373,68 @@ def evaluate(self, x, mu=None):
         return np.prod([f(x, mu) for f in self.functions], axis=0)
 
 
-class BitmapFunction(Function):
+class DataFieldFunction(Function):
+    """Define a 2D |Function| via a matrix that represents a data field.
+
+    Parameters
+    ----------
+    data_field
+        Data field as a 2D-|NumPy array|.
+    bounding_box
+        Lower left and upper right coordinates of the domain of the data field.
+    range
+        A pixel of value p is mapped to `(p / scaling_factor) * range[1] + range[0]`,
+        where scaling_factor is set below.
```

Comment:
Good point. We should probably add a remark for this, but an assertion is probably not needed?

Reply:
I guess at least there should be a remark that the `data_field` should have values between 0 and `scaling_factor` (if that is the intent). Also, "pixel" might not be the right word here, so replacing "A pixel of value p" by something like "A `data_field` value of p" would be good.
Otherwise, a more general mapping between any two intervals could be implemented.